iai_apartment/urdf/kitchen.xarco

```diff
@@ -1649,4 +1649,94 @@
 		</link>
 
 	</xacro:macro>
+
+<!-- extras -->
```

Comment:
Related to the "island" link, the origin of the link mesh is at the bottom, and it should be the center of the bounding box, such that it is in the middle of the Z dimension.

Reply:
Not following. Please elaborate.

Reply:
Please put the origin of the meshes package://iai_apartment/meshes/island/IAIKitchenIslandAreaTop.stl and package://iai_apartment/meshes/island/IAIKitchenIslandAreaTop.dae into the center of the bounding box using Blender: https://blender.stackexchange.com/questions/24078/set-origin-to-geometry-center

Reply:
There is no bounding box. The island is made of three pieces put together. I could provide you with a joint / empty link that points to the middle of the island, but changing the island_countertop's joint origin will just cause confusion.

Reply:
if it helps, I could create a collision mesh for the entire island. maybe excluding the front doors, drawers and handles, and including the back section's countertop edge.

Reply:
After talking in person, realized this is not needed.